### PR TITLE
Added Content-Type application/health+json to list of expected.

### PR DESCRIPTION
**Pull request by MTB90**
Added Content-Type application/health+json to list of expected based on:
* Draft: https://tools.ietf.org/html/draft-inadarei-api-health-check-03

### CODE REVIEW

**Comment by kingthorin** [comment]
Could you provide a reference link?

**Comment by MTB90** [comment]
> Could you provide a reference link?

No I see that this is draft: https://tools.ietf.org/html/draft-inadarei-api-health-check-03
so it should be probably rejected for now

**Comment by kingthorin** [comment]
Thanks. Draft RFCs often get implemented before ratification. I think I’m good with adding it.

**Review by kingthorin — APPROVED** [review]

File: docker/scripts/scripts/httpsender/Alert_on_Unexpected_Content_Types.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ var expectedTypes = [
 		"application/json",
 		"application/octet-stream",
 		"application/problem+json",
+		"application/health+json",
```
Comment (thc202) [inline docker/scripts/scripts/httpsender/Alert_on_Unexpected_Content_Types.js:14]: Could you move to the beginning of the array to keep them in alphabetic order?

**Comment by thc202** [comment]
Could you update the changelog?

File: docker/scripts/scripts/httpsender/Alert_on_Unexpected_Content_Types.js (hunk 1 above)
Comment (MTB90) [reply docker/scripts/scripts/httpsender/Alert_on_Unexpected_Content_Types.js]: Sure

**Comment by MTB90** [comment]
> Could you update the changelog?

Done

File: docker/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
 # Changelog
 All notable changes to the docker containers will be documented in this file.
 
+### 2020-10-12
+ - Added Content-Type application/health+json to list of expected
```
Comment (kingthorin) [inline docker/CHANGELOG.md:5]:
```suggestion
 - Alert_on_Unexpected_Content_Types.js > Added Content-Type application/health+json to the list of expected types.
```

**Review by kingthorin — CHANGES_REQUESTED** [review]
Could you please rebase this PR on an up-to-date `upstream/develop`?

After it's properly rebased we can change the base the PR is pointing to.

File: docker/CHANGELOG.md (hunk 2 above)
Comment (MTB90) [reply docker/CHANGELOG.md]: Rebased, updated base and updated CHANGELOG.md

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thanks!

**Review by psiinon — APPROVED** [review]

**Review by kingthorin — APPROVED** [review]
Looks good, thanks ⭐

**Pull request merged**